pushapkscript/docker.d/init_worker.sh

```diff
@@ -96,6 +96,8 @@ case $COT_PRODUCT in
         test_var_set 'GOOGLE_CREDENTIALS_FOCUS'
         test_var_set 'GOOGLE_PLAY_SERVICE_ACCOUNT_REFERENCE_BROWSER'
         test_var_set 'GOOGLE_CREDENTIALS_REFERENCE_BROWSER'
+        test_var_set 'GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_AURORA'
```

Comment:
Note to self: when we add new variables in new definitions, make sure these exist and are exposed in SOPS too.